Lab1.ipynb

```diff
@@ -0,0 +1,482 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.rcParams[\"figure.figsize\"] = (15,10)\n",
+    "fig, ax = plt.subplots(1, 1)\n",
+    "\n",
+    "x = np.linspace(0,10,1000);\n",
+    "prob = expon.pdf(x) \n",
+    "#ax.plot(x,prob,'r-',lw=5,alpha=0.6, label='expon pdf')\n",
+    "ax.plot(x, prob,'r-', lw=5, alpha=0.6, label='expon pdf')\n",
+    "\n",
+    "r = expon.rvs(size=100000)\n",
+    "ax.hist(r, density = True)\n",
+    "plt.show\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q3A"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Selecting my hypothetical measurement value to be 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# A little statistis Q3B"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Setting my hypothetical value to be 2. \n",
+    "If my signal free data has a exponential distribution, what is the probability that the data produces a signal that \n",
+    "is equally or more signal-like than the signal I observed which has a value of 2?\n"
```

Comment:
I think the hypothetical value you have chosen, 2, is when x = 2, and the signals we can observe from your plot in Q2A are all less than 0.6. The background signal at x = 2 is about 0.1 I think?